typing-react-components/Exercise1/problem/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta name="theme-color" content="#000000">
+	<!--
+      manifest.json provides metadata used when your web app is added to the
+      homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
+    -->
+	<link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+	<link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
```

Comment:
I think we can remove most of this boilerplate.

This is what I’ve included in my bit: https://github.com/bitovi/trainings/blob/807e57b9c0c9b385af99f60aed3aa23727a8bdb8/intro-to-accessibility-first/audio/solution/index.html#L1-L17